.github/workflows/pr.yaml

```
@@ -90,6 +93,16 @@ jobs:
       with:
         name: Workflow artifacts
         path: changed_repositories.list
+    - name: Get number of changed tools and compute number of chunks
+      id: get-chunks
+      run: |
+        planemo ci_find_tools --output changed_tools.list $(cat changed_repositories.list)
+        nchunks=$(wc -l changed_tools.list)
```

We need to not have the filename in the wc output
```suggestion
        nchunks=$(wc -l < changed_tools.list)
```